.github/workflows/run_tests.yml

```diff
@@ -103,18 +107,19 @@ jobs:
                       ${{ steps.pre-commit-cache.outputs.result }}-
 
             - name: Run Nox
-              run: nox --force-color --python=${{ matrix.python-version }}
+              run: |
```

Comment:
In ehrapy I've decided to put single line commands without a `|` and only if it's multiple lines to include the `|`. What do you think?

Reply:
Hm, guess for single lines, I'd prefer the version without `|` just like in ehrapy